clients/cli/src/main.rs

```diff
@@ -135,9 +135,14 @@ fn delete(core: &Core, target: &str, force: bool) -> Result<(), CliError> {
 
     if !force {
         let mut phrase = format!("delete '{}'", target);
-        if f.is_folder() {
-            let count = core.get_and_get_children_recursively(f.id)?.len();
-            phrase = format!("{phrase} and its {count} children")
+        let count = core
```

Comment:
could also save the recursive get children call for if we're actually dealing with a folder

Reply:
Good call